src/python/pants_release/release.py

```diff
@@ -452,17 +432,6 @@ def get_pgp_program_name() -> str:
     return configured_name or "gpg"
 
 
-@contextmanager
-def set_pants_version(version: str) -> Iterator[None]:
```

Comment:
Praise be. This has been a thorn in our side for so long. I think @chrisjrn still has PTSD from it.